_posts/2016-12-05-utah-sgid-statewide-roads-data-layer-updates.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Utah SGID Statewide Roads Data Layer Updates 12/05/2016'
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2016-12-05
+categories:
+- Data
+- Featured
+tags:
+- SGID
+- Data
+- Utah
+- Roads
+- Centerlines
+- Statewide
+- GIS
+---
+
+Updates were made recently to the SGID10.Transportation.Roads feature class that resides on the [Utah SGID ArcSDE database server]({{ "/data/how-to-connect-to-the-sgid-via-sde/" | prepend: site.baseurl }}).
+
+The updated Roads data is also available as shapefiles and file geodatabase files for download on the [SGID FTP site](ftp://ftp.agrc.utah.gov/UtahSGID_Vector/UTM12_NAD83/TRANSPORTATION/PackagedData/_Statewide/UtahRoadAndHighwaySystem/).
+
+Geocoding services and ArcGIS Server Applications & Web Services are now using the updated SGID10.Transportation.Roads feature class.
+
+The following highlights were updated:
```

Comment:
would this be better worded as something like `The following are highlights from this update:`